upup/models/cloudup/resources/addons/networking.weave/k8s-1.12.yaml.template

```diff
@@ -172,9 +172,17 @@ spec:
           resources:
             requests:
               cpu: 50m
+              {{- if .Networking.Weave.MemoryResources }}
+              memory: {{ .Networking.Weave.MemoryResources }}
+              {{else}}
               memory: 200Mi
+              {{- end }}
             limits:
+              {{- if .Networking.Weave.MemoryResources }}
+              memory: {{ .Networking.Weave.MemoryResources }}
+              {{else}}
```

Comment:
Could be wrong, but you may need to add `-` here for whitespace control. Not sure what the output looks like though.

Reply:
Ah, thanks. Didn't know about that. Fixing.

Reply:
Checked other templates and just did it how they've done it there. See https://github.com/kubernetes/kops/pull/7310/commits/ff20844db2bb4db951d8eb264803d45c06d3fc66.